app/src/main/java/pl/edu/zut/mad/appwizut2/fragments/AddBusChooseLineFragment.java

```diff
@@ -142,7 +143,7 @@ public void onClick(View v) {
         @Override
         protected List<BusLine> doInBackground(Void... params) {
             try {
-                String jsonStr = BusTimetableLoader.downloadFromAvrisApi("http://bus.avris.it/api/linie");
+                String jsonStr = BusTimetableLoader.downloadFromAvrisApi(HTTPLinks.BUS + "linie");
```

Comment:
If You're extracting this string to HTTPLinks, maybe extract whole url? (Same for "linia-%d" below)